spec/features/operator/rbac/rbac_triggers_test.go

```diff
@@ -0,0 +1,17 @@
+package rbac
+
+import (
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestTriggersSA(t *testing.T) {
+	Convey("Given the Operator is installed", t, func() {
+		Convey("create a new namespace", func() {
+			Convey("Then I Create a new Triggers SA with Required RBAC", nil)
```

Comment:
Could you rephrase this? 
The test seems quite confusing

Reply:
Makes sense Created a new Test for that